### PR TITLE
Updated HQ description for 255 char limit for standalone HQ

### PR DESCRIPTION
**Pull request by v-dvedak**
Required items, please complete
   
   Change(s):
   - Updated HQ description for 255 char limit for standalone HQ

   Reason for Change(s):
   - HQ description exceeding 255 char limit for standalone HQ

   Version Updated:
   - Yes
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes/No/Need Help

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Hunting Queries/W3CIISLog/ClientIPwithManyUserAgents.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 id: 4edbb420-2df7-4089-9906-c335f065803e
 name: Same IP address with multiple csUserAgent
 description: |
+  'Alerts when same client IP connects with 1-15 different useragent strings in less than 1 hour. Limited to 50 or less connections. May indicate malicious activity. References: IIS status code mapping and Win32 Status code mapping.'
```
Comment (rahul0216) [inline Hunting Queries/W3CIISLog/ClientIPwithManyUserAgents.yaml:4]: No need of reference part here, rather extend the description itself.

File: Hunting Queries/W3CIISLog/RareClientFileAccess.yaml [hunk 2]
```diff
@@ -1,6 +1,8 @@
 id: a787a819-40df-4c9f-a5ae-850d5a2a0cf6
 name: URI requests from single client
 description: |
+  'Query finds connections to server files requested by only one client. Effective when actor uses static operational IP addresses. Threshold adjustable. Larger execution window increases reliability of results.'
```
Comment (rahul0216) [inline Hunting Queries/W3CIISLog/RareClientFileAccess.yaml:4]: Description looks broken. Can you rephrase it better?

File: Hunting Queries/W3CIISLog/SuspectedMailBoxExportHostonOWA.yaml [hunk 3]
```diff
@@ -1,6 +1,8 @@
 id: a523786c-8382-4029-80e9-f1a7ecd067c1
 name: Suspect Mailbox Export on IIS/OWA
 description: |
+  'Query finds suspicious files accessed on IIS server indicating exfiltration hosting. Observed when exporting mailbox files from OWA servers. Reference: Dark Halo Leverages SolarWinds Compromise.'
```
Comment (rahul0216) [inline Hunting Queries/W3CIISLog/SuspectedMailBoxExportHostonOWA.yaml:4]: Again, No need of reference string here.

**Review by rahul0216 — COMMENTED** [review]
Please address the inline comments.

File: Hunting Queries/W3CIISLog/SuspectedMailBoxExportHostonOWA.yaml (hunk 3 above)
Comment (v-dvedak) [reply Hunting Queries/W3CIISLog/SuspectedMailBoxExportHostonOWA.yaml]: Removed reference string and updated description.

File: Hunting Queries/W3CIISLog/ClientIPwithManyUserAgents.yaml (hunk 1 above)
Comment (v-dvedak) [reply Hunting Queries/W3CIISLog/ClientIPwithManyUserAgents.yaml]: Removed reference string and updated description.

File: Hunting Queries/W3CIISLog/RareClientFileAccess.yaml (hunk 2 above)
Comment (v-dvedak) [reply Hunting Queries/W3CIISLog/RareClientFileAccess.yaml]: Updated description.

**Review by rahul0216 — APPROVED** [review]
Changes look good.

**Review by v-atulyadav — APPROVED** [review]
